src/usng.ts

```diff
@@ -638,15 +638,27 @@ extend(Converter.prototype, {
     }
   },
 
+  isInUPSSpace(lat) {
```

Comment:
Since this is a typescript file we could define the type for this function to catch possible issues at compile time.